python/kwiver/vital/algo/metadata_map_io.cxx

```diff
@@ -0,0 +1,55 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#include <python/kwiver/vital/algo/metadata_map_io.h>
+#include <python/kwiver/vital/algo/trampoline/metadata_map_io_trampoline.txx>
+
+#include <pybind11/pybind11.h>
+
+namespace kwiver {
+
+namespace vital {
+
+namespace python {
+
+namespace py = pybind11;
```

Comment:
FYI, pedantically, this alias already exists from the header and is therefore redundant. (It looks like the existing bindings have the same "issue", though...)

Reply:
Is it worth leaving this in for the sake of clarity? Alternatively, should I address all of these on a separate PR?

Reply:
Might be worth a separate PR, might not be worth bothering at all. It's not *harmful*, really, just superfluous. As far as this PR, it's consistent with what we have already, so *if* we do anything about it, doing so in a new PR may be best.

That said, if you're going to look at addressing it at all, it's perhaps arguable if we really want `kwiver::vital::python::py → pybind11` to be public? (Although OTOH I'm not sure the Python binding headers exactly count as "public", since I guess they probably aren't used except to generate Python bindings?)

Reply:
Well in any case I won't address it here then. I'm not sure, but I believe you're correct that these files are only used to generate bindings so it seems like a nonissue.

Reply:
I don't think we want to be exposing `kwiver::vital::python::py`  because, exactly as Matt said, those headers are used only for binding generation, as well as clarity , and because the `py` qualifier should, both here and the header, be declared outside of the `kwiver::vital::python` scope. That being said in another PR, I don't really have a problem with a potential refactor, as it is redundant. That being said, leaving it makes a refactor toward algo abandoning the use of headers that much more painless. Conversation for another day/PR.

Reply:
I updated the ordering in both places.